include/api/CAnomalyJob.h

```diff
@@ -424,6 +426,10 @@ class API_EXPORT CAnomalyJob : public CDataProcessor {
     //! Object to which the output is passed
     CJsonOutputWriter m_JsonOutputWriter;
 
+    //! Configuration settings for the analysis parsed from
+    //! JSON configuration file.
```

Comment:
It would be good to say why this is non-const.  It's because it gets updated by job updates right?  And of course since it's a reference that will also affect everywhere else that has a reference.